dist/Readme

```diff
@@ -0,0 +1,3 @@
+You may use the service files provided here to run hercules via systemd.
+Make sure to replace the executable path and working directory according to your installation and copy the files to the appropriate location on your system.
+The files should take care of starting and stopping the hercules server and monitor together.
```

Comment:
>The files should take care of starting and stopping the hercules server and monitor together.


Do we document the inter-dependencies? The hercules-server relies on the monitor to get paths, but why would it be an issues for example to restart the hercules-server without also restarting the monitor? The monitor doesn't reconnect?